backend/routes/system_routes.py

```
@@ -0,0 +1,336 @@
+    t0 = time.monotonic()
+    try:
+        import redis.asyncio as aioredis
+        r = aioredis.Redis(host=settings.REDIS_HOST, port=settings.REDIS_PORT, db=0, socket_timeout=2)
+        await r.ping()
+        info = await r.info("memory")
+        used = info.get("used_memory_human", "?")
+        max_mem = info.get("maxmemory_human", "unlimited")
+        await r.aclose()
+        ms = round((time.monotonic() - t0) * 1000, 1)
+        return ServiceHealth(status="up", latency_ms=ms, details=f"memory: {used} / {max_mem}")
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)